src/lib/orderModification.ts

```diff
@@ -1,5 +1,6 @@
 import Abacus, { Nullable } from '@dydxprotocol/v4-abacus';
 import { OrderExecution } from '@dydxprotocol/v4-client-js';
+// ts-ignore
```

Comment:
whats the error here?

Reply:
oops, it's fixed now, I forgot I added the ignore. Thank youuuu